motionplan/motionPlanner_test.go

```diff
@@ -398,17 +400,18 @@ func TestArmAndGantrySolve(t *testing.T) {
 		nil,
 	)
 	test.That(t, err, test.ShouldBeNil)
-	solvedPose, err := fs.Transform(newPos[len(newPos)-1], frame.NewPoseInFrame("xArmVgripper", spatialmath.NewZeroPose()), frame.World)
+	solvedPose, err := fs.Transform(plan[len(plan)-1], frame.NewPoseInFrame("xArmVgripper", spatialmath.NewZeroPose()), frame.World)
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, spatialmath.PoseAlmostCoincidentEps(solvedPose.(*frame.PoseInFrame).Pose(), goal1, 0.01), test.ShouldBeTrue)
+	visualization.VisualizePlan(fs, fs.Frame("xArmVgripper"), plan, &frame.WorldState{})
```

Comment:
👀

Reply:
Yeah.  I actually have a lot of stuff that is going to change with the merge into main.  Was about to message about this